addons/binding/org.openhab.binding.opensprinkler/src/main/java/org/openhab/binding/opensprinkler/internal/api/OpenSprinklerGpioApi.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2014-2015 openHAB UG (haftungsbeschraenkt) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.opensprinkler.internal.api;
+
+import static org.openhab.binding.opensprinkler.internal.api.OpenSprinklerApiConstants.*;
+
+import com.pi4j.io.gpio.GpioController;
+import com.pi4j.io.gpio.GpioFactory;
+import com.pi4j.io.gpio.GpioPinDigitalOutput;
+
+/**
+ * The {@link OpenSprinklerGpioApi} class is used for communicating with
+ * the OpenSprinkler PI when OpenHAB is installed on the same Raspberry PI
+ * that the OpenSprinkler PI device is using.
+ *
+ * @author Jonathan Giles
```

Comment:
you have to mention him on your commit message with an "Also-By" footer, see https://wiki.eclipse.org/Development_Resources/Handling_Git_Contributions#Multiple_Authors